src/soundjs/soundjs.js

```diff
@@ -19,29 +19,22 @@
 
 (function() {
 
-    if (typeof(CerosSDK) === "undefined") {
-        var sdkScript = document.createElement('script');
-        sdkScript.type = "text/javascript";
-        sdkScript.async = true;
-        sdkScript.onload = activatePlaySound;
-        sdkScript.src = "//sdk.ceros.com/standalone-player-sdk-v3.js";
+    require.config({
 
-        document.getElementsByTagName('head')[0].appendChild(sdkScript);
-    } else {
-        activatePlaySound();
-    }
+        shim: {
+            SoundJS: {
+                exports: 'createjs'
+            }
+        },
 
-    function loadSoundJs() {
-        var soundScript = document.createElement('script');
-        soundScript.type = "text/javascript";
-        soundScript.async = true;
-        soundScript.src = "//code.createjs.com/soundjs-0.6.2.min.js";
-        document.body.appendChild(soundScript);
-    }
-
-    function activatePlaySound() {
-        loadSoundJs();
+        paths: { 
+            CerosSDK: "//sdk.ceros.com/standalone-player-sdk-v3",        
+            SoundJS: "https://code.createjs.com/soundjs-0.6.2.min",
+        }
+        
+    });
 
+    require(['CerosSDK', 'SoundJS'], function (CerosSDK, createjs) {
```

Comment:
Weird that the SoundJS variable is called `createjs` here. Since this can be anything, `SoundJS` would be more accurate.